admin-dev/themes/new-theme/js/app/pages/stock/components/overview/products-actions.vue

```diff
@@ -83,10 +85,11 @@
         return this.$store.getters.selectedProductsLng;
       },
     },
+    mixins: [TranslationMixin],
     watch: {
       selectedProductsLng(value: number): void {
         if (value === 0 && this.$refs['bulk-action']) {
-          this.$refs['bulk-action'].checked = false;
+          (<any> this.$refs['bulk-action']).checked = false;
```

Comment:
```suggestion
          (<HTMLInputElement> this.$refs['bulk-action']).checked = false;
```
Maybe you can use this one instead to be more accurate?